scripts/docker-quick-start/Dockerfile.apollodb

```diff
@@ -0,0 +1,36 @@
+#
```

Comment:
<img width="653" alt="image" src="https://user-images.githubusercontent.com/33855684/160957772-6b0b75f4-0c54-4ac8-8139-4016f7f631c4.png">

I used this Dockerfile to build the two components. For the end user, the images are shipped with the application template, so they don't need to use this Dockerfile.

Thank you for pointing out the carelessness in my documentation. 😂

Reply:
I see. Is it possible to include this docker file as part of Rainbond apollo template instead of adding it to apollo repository? What concerns me is that it's not used in any of apollo deployments, so there is no guarantee this file would be correct in the future, or even maybe deleted by some one mistakenly.

Reply:
OK, I understand your concern, I will append a commit to delete this Dockerfile, it will only stored in the application template.

Reply:
done～